publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/CLI.java

```diff
@@ -146,29 +146,36 @@ public void handleContent(String content) {
             } else {
                 throw new Exception();
             }
+        }catch (RemRemPublishException e) {
+            JsonArray jarray = new JsonArray();
+            PublishResultItem result = new PublishResultItem(null, 404,
+                 null,"Exchange " + e.getMessage());
+            jarray.add(result.toJsonObject());
+            System.out.println(new GsonBuilder().setPrettyPrinting().create().toJson(jarray));
```

Comment:
can we add logger here instead of sysout?
why are we printing the json here? any purpose